src/ScnSocialAuth/Controller/UserController.php

```diff
@@ -38,14 +38,20 @@ class UserController extends AbstractActionController
     protected $redirectCallback;
 
     /**
+     * @var ZfcUserModuleOptions
+     */
+    protected $zfcModuleOptionsç
```

Comment:
There's no semi-colon at the end of this: https://travis-ci.org/SocalNick/ScnSocialAuth/jobs/140440270#L436